pennylane/vqe/vqe.py

```diff
@@ -120,6 +120,35 @@ def __str__(self):
 
         return "\n+ ".join(terms)
 
+    @property
```

Comment:
@Lucaman99 I _might_ suggest making this a method rather than a property. I usually expect properties not to perform (expensive) logic with every call; a method better indicates to the user that logic could be occurring with every call.

If a property is expensive to calculate, or has side effects, then it likely would be much better as a normal method. Making it look like an attribute access means that developers will likely think of it as an attribute access, which is cheap and non-destructive; this would be a rich source of bugs or performance problems.